packages/dds/tree/src/changeset/fence.json

```diff
@@ -1,5 +1,5 @@
 {
     "tags": [ "changeset" ],
     "exports": [ "index"],
-    "imports": ["schema","tree","util"]
-}
\ No newline at end of file
+    "imports": ["rebase", "schema","tree","util"]
```

Comment:
I feel like it makes sense for rebase to depend on changeset (since it needs to use changeset types), so adding this dep will cause a cycle if we do that.

Reply:
Maybe put ProgressiveEditBuilder in transaction instead, since it can depend on both, and that what will use it?

Reply:
I can move it back for now, but it seems like ChangeFamily should eventually be independent of transaction.